tools/kubernetes/docker/spark-block-cleaner/entrypoint.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+#
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+# entrypoint for spark-block-cleaner
+
+# shellcheck disable=SC2046
+exec /usr/bin/tini -s -- "java -cp ${CLASS_PATH}:${CLEANER_CLASSPATH}
+  org.apache.kyuubi.tools.KubernetesSparkBlockCleaner
+  | tee /log/cleanerLog/cleaner$(date "+%Y%m%d%H%M%S").out"
```

Comment:
` 2>&1 > /log/cleanerLog/cleaner$(date "+%Y%m%d%H%M%S").out` ?

Reply:
In this case, we need both file log and kubectl log. So use tee to get two outStream.